Gemfile.lock

```diff
@@ -448,22 +447,20 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    omniauth_openid_connect (0.7.1)
+    omniauth_openid_connect (0.6.1)
```

Comment:
Weird. This downgrade happened on several dependabot PRs.

Reply:
I noticed #12109 bumped `openid_connect` to 2.3.

But shouldn't our gemfile prevent updates of `openid_connect` to v2?
> gem 'openid_connect', '~> 1.3'

Is that the reason dependabot is downgrading us again?

Reply:
Good pickup! I'm wondering why we require that version of openid_connect. Do we need to go back or can we upgrade?

Reply:
You are brilliant, as always, @sigmundpetersen. Now we got a solution:

* https://github.com/openfoodfoundation/openfoodnetwork/pull/12131